docs/source/interactive/python-ipython-diff.rst

```diff
@@ -241,9 +240,10 @@ Magics with double percent signs (``%%``) can spread over multiple lines, but th
     map auto_home    0Bi    0Bi    0Bi   100%        0       0  100%   /hom
 
 
-Combining it all
-----------------
+..
+   Combining it all
+   ----------------
 
-::
+   ::
 
-    find a snippet that combine all that into one thing!
+       find a snippet that combine all that into one thing!
```

Comment:
To be honest, I'd rather just remove this whole section. It would be pretty hard to find a good, original use scenario that is concise and clear enough to be put there.

Reply:
Fair enough. See 55d4c79f1.